### ISSUE TITLE
File chooser component lacks keyboard navigation support

### ISSUE THREAD

**Issue by Venefilyn** [cockpit-project/cockpit#23704]
Within the file chooser there is keyboard support for going up and down with arrow keys, and using tab to go between rows. However there is lackluster support for the navigation overall and does move focus when rows are removed from navigating to another directory or with file location breadcrumbs going a directory up
